### PR TITLE
Let `pacman-helper.sh` run in Linux

### PR DESCRIPTION
**Pull request by dscho**
The idea is of course to help https://github.com/git-for-windows/git-for-windows-automation/pull/86

### CODE REVIEW

File: pacman-helper.sh [hunk 1]
```diff
@@ -57,7 +53,16 @@ upload)
 	;;
 esac
 
-this_script_dir="$(cygpath -am "${0%/*}")"
+this_script_dir="$(case "$(uname -s)" in
+	MSYS|MINGW*)
+		# MSYS2's mingw-w64-$arch-ca-certificates seem to lag behind ca-certificates
+		CURL_CA_BUNDLE=/usr/ssl/certs/ca-bundle.crt
+		export CURL_CA_BUNDLE
```
Comment (rimrul) [inline pacman-helper.sh:60]: Doesn't the `$()` command substitution run in a subshell? Wouldn't that make the export useless?

Comment (dscho) [reply pacman-helper.sh:60]: The `export` does not run in a subshell, but its effect extends to all subshells, so it's very useful: every `curl` call in this script, as well as in the `wingit-snapshot-helper.sh` script, will now heed that path.

Comment (rimrul) [reply pacman-helper.sh:60]: Isn't this whole hunk one big subshell?

Comment (dscho) [reply pacman-helper.sh]: Ouch. You're right. Let me fix that.

Comment (dscho) [reply pacman-helper.sh]: How does [this](https://github.com/git-for-windows/build-extra/compare/ae824da594a500309b3c67cfe30df044003f369b..78ef4e3cda962813a5520f76403b4e189fd89676) look to you, @rimrul?

**Review by rimrul — APPROVED** [review]

**Pull request merged**